packages/bsync/proto/bsync.proto

```diff
@@ -67,6 +67,44 @@ message ScanNotifOperationsResponse {
 }
 
 
+enum Method {
+  METHOD_UNSPECIFIED = 0;
+  METHOD_CREATE = 1;
+  METHOD_UPDATE = 2;
+  METHOD_DELETE = 3;
+}
+
+message Operation {
+  string id = 1;
+  string collection = 2;
+  string actor_did = 3;
+  string rkey = 4;
```

Comment:
Small thought here.  If we're trying to evoke the structure of repositories with did/collection/rkey, we might consider the actors the top level of the hierarchy and list it first.
```suggestion
  string actor_did = 2;
  string collection = 3;
  string rkey = 4;
```